src/app/[locale]/(app)/analytics/page.tsx

```
@@ -0,0 +1,574 @@
+'use client'
+
+import dynamic from 'next/dynamic'
+import { useSearchParams, usePathname, useRouter } from 'next/navigation'
+import React, { useState, useCallback, useEffect } from 'react'
```

ESLint

[error] 'useEffect' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)